src/router/Router.tsx

```diff
@@ -11,6 +11,7 @@ function Router() {
       <Routes>
         <Route path={"/"} Component={LoginPage} />
         <Route path={"/home"} Component={HomePage} />
+        <Route path={"/objectIdentifierPage"} Component={objectIdentifierPage} />
```

Comment:
```suggestion
        <Route path={"/object-identifier"} Component={objectIdentifierPage} />
```
Don't you think this is good?

Reply:
fixed https://github.com/Arutha-Project/Arutha-Frontend/pull/20/commits/c72b4e68e2c66cbbc6e67a3b72a788f4da734b0a